demo/main.ts

```diff
@@ -66,14 +68,16 @@ Promise.all([
 	fetch(`${MCMETA}registries/item/data.min.json`).then(r => r.json()),
 	fetch(`${MCMETA}summary/assets/block_definition/data.min.json`).then(r => r.json()),
 	fetch(`${MCMETA}summary/assets/model/data.min.json`).then(r => r.json()),
+	fetch(`${MCMETA}summary/data/item_definition/data.min.json`).then(r => r.json()),
```

Comment:
btw this should probably be moved to `summary/assets/item_definition`

Reply:
Fixed!